synapse/cortex.py

```diff
@@ -4257,6 +4279,135 @@ def _initCortexHttpApi(self):
         self.addHttpApi('/api/v1/axon/files/by/sha256/([0-9a-fA-F]{64}$)', CortexAxonHttpBySha256V1, {'cell': self})
         self.addHttpApi('/api/v1/axon/files/by/sha256/(.*)', CortexAxonHttpBySha256InvalidV1, {'cell': self})
 
+        self.addHttpApi('/api/ext/(.*)', s_httpapi.ExtApiHandler, {'cell': self})
+
+    def _initCortexExtHttpApi(self):
+        self._exthttpapis.clear()
+
+        byts = self.slab.get(self._exthttpapiorder, self.httpextapidb)
+        if byts is None:
+            return
+
+        order = s_msgpack.un(byts)
+
+        for iden in order:
+            byts = self.slab.get(s_common.uhex(iden), self.httpextapidb)
+            if byts is None:  # pragma: no cover
+                logger.error(f'Missing HTTP API definition for iden={iden}')
+                continue
+            self._exthttpapis.append(s_msgpack.un(byts))
+
+    async def addHttpExtApi(self, adef):
+        adef['iden'] = s_common.guid()
+        adef = s_schemas.HttpExaAPIConfSchema(adef)
```

Comment:
If this is a function it should start with lower camel case. Also not sure we need the word Schema in every schema validator callback :)